index.css

```diff
@@ -0,0 +1,46 @@
+@import url("https://fonts.googleapis.com/css2?family=Dongle:wght@400;700&family=Lato:wght@400;700&family=Noto+Sans:wght@400;700&family=Roboto:wght@300;400;500;700&display=swap");
```

Comment:
We can keep this in `index.html` head tag.

Reply:
Updated and pushed to github.